gax-grpc/src/main/java/com/google/api/gax/grpc/OperationFuture.java

```diff
@@ -29,58 +29,117 @@
  */
 package com.google.api.gax.grpc;
 
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.api.core.ApiFunction;
 import com.google.api.core.ApiFuture;
+import com.google.api.core.ApiFutures;
 import com.google.api.core.BetaApi;
+import com.google.api.gax.retrying.RetryingFuture;
 import com.google.longrunning.Operation;
 import com.google.protobuf.Message;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.Executor;
+import java.util.concurrent.Future;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 
 /** An ApiFuture which polls a service through OperationsApi for the completion of an operation. */
 @BetaApi
-public final class OperationFuture<ResponseT extends Message> implements ApiFuture<ResponseT> {
+public final class OperationFuture<ResponseT extends Message, MetadataT extends Message>
+    implements ApiFuture<ResponseT> {
+  @Override
+  public String toString() {
+    return pollingFuture.toString();
+  }
```

Comment:
Oops, this is a debugging artifact which I forgot to remove. Removed completely.